libr/core/cmd_quit.c

```diff
@@ -10,26 +10,44 @@ static int cmd_quit(void *data, const char *input) {
 		"q!!","","force quit without saving history",
 		"q"," 1","quit with return value 1",
 		"q"," a-b","quit with return value a-b",
+		"q[y/n][y/n]","","quit, chose to kill process, chose to save project ",
 		NULL};
 	if (input)
 	switch (*input) {
 	case '?':
 		r_core_cmd_help (core, help_msg);
 		break;
 	case '!':
-		if (input[1] == '!')
+		if (input[1] == '!') {
 			r_config_set (core->config, "scr.histsave", "false");
+		}
 		core->num->value = -1;
 		return -2;
 	case '\0':
 		core->num->value = 0LL;
 		return -2;
 	default:
-		if (*input == ' ')
+		if (*input == ' ') {
```

Comment:
Use r_str_chop_ro instead of manually skipping spaces

> On 10 Jul 2016, at 18:28, Jeffrey Crowell <notifications@github.com> wrote:
> 
> In libr/core/cmd_quit.c:
> 
> >  		core->num->value = -1;
> >  		return -2;
> >  	case '\0':
> >  		core->num->value = 0LL;
> >  		return -2;
> >  	default:
> > -		if (*input == ' ')
> > +		if (*input == ' ') {
> Should this be in a while loop to get first non space?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>